Data Files/MWSE/mods/CraftingFramework/components/Recipe.lua

```diff
@@ -16,6 +16,7 @@ local MaterialRequirementSchema = {
 }
 
 
+---@class Recipe : craftingFrameworkRecipe
```

Comment:
This trick will hide: 
- `schema`
- `.getRecipe`
- `:new`
- `registeredRecipes`
from objects constucted with `Recipe:new()`. I can remove this if you want.